File: wlr/SPX.Entities/CoordVerifier.cs
using System.Linq;
using SPX.Core;
using SPX.Sprites;
using SPX.Entities;

namespace SPX.Entities
{
    public static class CoordVerifier
    {
        public static bool IsValid(Point2 position)
        {
            return (position.PosX >= 0 && position.PosY >= 0 && position.PosX < XnaManager.WindowWidth && position.PosY < XnaManager.WindowHeight);
        }

        public static bool IsBlocked(Point2 target)
        {
            return EntityManager.IsLocationBlocked(target);
        }   

        public static bool Contains(Point2 target, int type)
        {
            return EntityManager.AnyContains(target, type);
        }
    }
}

File: proj/src/spx/entities/CoordVerifier.java
package spx.entities;

import spx.bridge.EntityType;
import spx.core.Point2;
import spx.core.SpxManager;

public class CoordVerifier {
    public static boolean isValid(Point2 position) {
        return (position.PosX >= 0 && position.PosY >= 0 && position.PosX < SpxManager.WindowWidth && position.PosY < SpxManager.WindowHeight);
    }

    public static boolean isBlocked(Point2 target)

    {
        return EntityManager.isLocationBlocked(target);
    }

    public static boolean contains(Point2 target, EntityType type)

    {
        return EntityManager.anyContains(target, type);
    }
}
